### PR TITLE
docs: change chatroom from gitter to discord

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes https://github.com/inveniosoftware/opensource/issues/67

### CODE REVIEW

**Comment by FlorianCassayre** [comment]
Thanks! I was going to include it in inveniosoftware/invenio#4046 but you did it. Just beware that one of us might have to rebase on the other.

Issue should be safe to close after that. (unless I missed something?)

File: docs/community/channels.rst [hunk 1]
```diff
@@ -14,8 +14,8 @@ Chatrooms
 ---------
 Most day-to-day communication is happening in our chatrooms:
 
-- `Public chatroom <https://gitter.im/inveniosoftware/invenio>`_ (for everyone)
-- `Developer chatroom <https://gitter.im/inveniosoftware/invenio>`_ (for
+- `Public chatroom <https://discord.gg/8qatqBC>`_ (for everyone)
+- `Developer chatroom <https://discord.gg/8qatqBC>`_ (for
```
Comment (FlorianCassayre) [inline docs/community/channels.rst:18]: Weird two have the same link twice (though it was already this way).

File: README.rst [hunk 2]
```diff
@@ -21,8 +21,8 @@
 .. image:: https://travis-ci.org/inveniosoftware/invenio.svg?branch=master
     :target: https://travis-ci.org/inveniosoftware/invenio
 
-.. image:: https://badges.gitter.im/Join%20Chat.svg
-    :target: https://gitter.im/inveniosoftware/invenio
+.. image:: https://img.shields.io/badge/Discord-join%20chat-%237289da
```
Comment (FlorianCassayre) [inline README.rst:24]: :+1:

**Review by FlorianCassayre — APPROVED** [review]

File: docs/community/channels.rst (hunk 1 above)
Comment (diegodelemos) [reply docs/community/channels.rst:18]: Let's list only one link then. Later on, once Discord evolves and new chats are created, we can extend this section:
```diff
+ - `Public chatroom <https://discord.gg/8qatqBC>`_ (for everyone)
- - `Developer chatroom <https://discord.gg/8qatqBC>`_ (for
```

Could you also rebase against latest master?

**Pull request merged**